Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:3
+
+# Setup env
+ENV NODE_OPTIONS=--openssl-legacy-provider
+
+# Install Node and npm
+RUN apk add --update autoconf automake build-base libtool nasm pkgconf nodejs npm git && rm -rf /var/cache/apk/*
+
+# Change workdir to have separate place for app
+WORKDIR /gitops-website
+
+# Add required files
+ADD . .
+
+# Install dependencies
+RUN npm i
+
+# Expose port for server
+EXPOSE 8000
+

```

Comment:
Talked to @samueltauil about this. I would rather not run it as root if at all possible.

I would suggest adding something like

```
RUN chown -R 1001:0 /gitops-website && chmod -R ug+rwx /gitops-website
USER 1001
```

Reply:
Maybe we can even use `node:16-alpine`?